### ISSUE TITLE
Negative value result in Chapter 12

### ISSUE THREAD

**Issue by akrsnr** [markjprice/cs10dotnet6#16]
I'm trying your recorder.cs example on my Mac M1 with Rider of Jetbrains. I get the following output after several runs.

```
Processing. Please wait...
**-1,867,776 physical bytes used.**
14,041,088 virtual bytes used.
00:00:06.0042556 time span ellapsed.
6,004 total milliseconds ellapsed.
```

Can we say there exists a bug in .Net 6? When I'm running it on parallel desktop, it works as illustrated in your book.

**Comment by markjprice** [comment]
Yes, it could be a bug related to some CPU difference with the M1. I will try to find time to investigate so I will leave this issue open for others to see.

**Comment by markjprice** [comment]
I have added an erratum for this issue: https://github.com/markjprice/cs10dotnet6/blob/main/errata.md#page-510---implementing-a-recorder-class
